migrations/20170605161512_rename-followers.js

```diff
@@ -1,11 +1,13 @@
 exports.up = function (knex, Promise) {
   return knex.schema.table('communities_tags', table => {
-    table.integer('followers').defaultTo(0)
+    table.dropColumn('followers')
```

Comment:
hmm. Yes. My database and migrations got a little messed up somewhere along the way I think. Removed the references to 'followers' now.